### DISCUSSION TITLE
(mini.notify) How to disable specific notification?

### DISCUSSION THREAD

**Discussion by mainendra** [nvim-mini/mini.nvim#664] [Q&A] [answered]
Is there a way we can disable specific notification? Actually I want to disable `null-ls` notifications but allow other LSP server notifications like `tsserver`

**Comment by echasnovski** [comment, marked as answer]
Right now the suggested approach is to utilize `content.sort` to modify which notifications will be shown. As it technically [has complete control over what and in which order can be shown](https://github.com/echasnovski/mini.nvim/blob/2e2931b3822bf357d2f69cbc0bcb2d99a3f0d4c8/doc/mini-notify.txt#L136-L139).

One downside is that it has to rely only on what is [supplied](https://github.com/echasnovski/mini.nvim/blob/2e2931b3822bf357d2f69cbc0bcb2d99a3f0d4c8/doc/mini-notify.txt#L65-L73), and realistically it is only message and level (as the only documented inputs to `vim.notify()`).

Here is what I currently have [to not show "Diagnosing..." notifications from `lua_ls`](https://github.com/echasnovski/nvim/blob/06e32c2ad6493815c09d8dbb2e3042aa0eee086f/init.lua#L36-L45).
